Configuration/Testing/Behat/Settings.Mvc.yaml

```diff
@@ -0,0 +1,8 @@
+Neos:
+  Flow:
+    mvc:
+      routes:
+        'Neos.Neos':
+          variables:
+            # set explicitly for testing as it may be overriden by other packages (e.g. Neos.Demo)
+            'defaultUriSuffix': '.html'
```

Comment:
This breaks Neos behat tests currently:
![image](https://github.com/neos/redirecthandler-neosadapter/assets/307571/3e4832f0-04e0-4c8e-8192-a58eb503d48b)

Reply:
This can be tackled with #4389 hopefully

Reply:
https://github.com/neos/neos-development-collection/issues/4389